apis/validator/proposer_config.yaml

```
@@ -0,0 +1,24 @@
+get:
+  tags:
+    - ValidatorRequiredApi
+    - Validator
+  operationId: "proposerConfig"
+  summary: "Produce a proposer config"
+  description: |
+    Requests the beacon node or DVT middleware to produce the proposer config. The proposer_config enables validator registrations
```

```suggestion
    Requests the beacon node or DV middleware to produce the proposer config. The proposer_config enables validator registrations
```

We should align with Brett on this brand wide, but I prefer DV most of the time 😅